epictrack-web/src/components/workPlan/event/EventForm.tsx

```diff
@@ -127,7 +119,13 @@ const EventForm = ({ onSave, event, isFormFieldsLocked }: EventFormProps) => {
   } = methods;
 
   React.useEffect(() => {
-    if (selectedConfiguration && selectedConfiguration.multiple_days) {
+    if (
+      selectedConfiguration &&
+      selectedConfiguration.event_category_id === EventCategory.EXTENSION
+    ) {
+      setAnticipatedLabel("Anticipated Order Date");
+      setActualDateLabel("Actual Order Date");
+    } else if (selectedConfiguration && selectedConfiguration.multiple_days) {

```

Comment:
let's use chaining, selectedConfiguraion?.event_category_id === EventCategory.EXTENSION instread of having selectedConfiguration && at the start